### PR TITLE
Update rundir configuration files to point to corrected NEI day-of-week scale factors (Closes #1608)

### LINKED ISSUE #1608: Possible issue with NEI Day of Week (DOW) scalars
**Description of the problem**

I have been investigating a problem where my simulation results were noticeably different within NEI-covered areas depending on the date I started my simulation.  An example of this is shown in the attached figure (NO2Difference.png):

NO2Difference

Here I started one simulation in mid-May 2007 and a second simulation using the first simulation's restart file for June 14.  Comparing the output from these two simulations on June 30 demonstrates the sort of differences I am seeing.

**Description of troubleshooting performed**

Disabling the DOW scalar within HEMCO removed the inconsistencies between my simulations with different start dates, suggesting that the day of week indices is not being calculated and/or applied correctly to the monthly mean NEI emissions.  Closer inspection of the NEI99 Day of Week (DOW) scalars (NEI99.dow.geos.1x1.nc) also showed that all values are 1 for weekdays and <1 for weekends (see below).  As a result, it appears that even if the scalars were being appropriately applied they would erroneously produce a decrease in the NEI monthly total emissions compared to the original monthly totals.

NEI99DOWscalars

**GEOS-Chem version**

13.2.1

**Description of modifications**

No longer applying the NEI99 DOW scalars to NEI2011 removes the immediate issue, but there may be a remaining issue within GEOS-Chem as to how such DOW scalars are implemented.

### PR DESCRIPTION
**Pull request by yantosca**
This is the companion PR to #1608.  Eloise Marais (@eamarais) has created a corrected day-of-week scale factor file for the NEI emissions.  We have updated the relevant `HEMCO_Config.rc` and `ExtData.rc` rundir template files to point to the corrected file.

NOTE: This should be a zero-diff update w/r/t the fullchem benchmark simulation, as the NEI emissions are disabled by default.  This will only affect users of the NEI11 or NEI16 emissions.

### CODE REVIEW

**Comment by yantosca** [comment]
Integration tests are in progress

**Comment by yantosca** [comment]
All GEOS-Chem Classc integration tests passed:
```console
==============================================================================
GEOS-Chem Classic: Execution Test Results

Using 24 OpenMP threads
Number of execution tests: 51
==============================================================================
 
Execution tests:
------------------------------------------------------------------------------
gc_025x03125_CH4_geosfp_47L_na......................Execute Simulation....PASS
gc_025x03125_fullchem_geosfp_47L_na.................Execute Simulation....PASS
gc_05x0625_CH4_merra2_47L_na........................Execute Simulation....PASS
gc_05x0625_fullchem_merra2_47L_na...................Execute Simulation....PASS
gc_2x25_aerosol_merra2..............................Execute Simulation....PASS
gc_2x25_carbon_merra2...............................Execute Simulation....PASS
gc_2x25_CH4_merra2..................................Execute Simulation....PASS
gc_2x25_CO2_merra2..................................Execute Simulation....PASS
gc_2x25_fullchem_merra2.............................Execute Simulation....PASS
gc_2x25_Hg_merra2...................................Execute Simulation....PASS
gc_2x25_metals_merra2...............................Execute Simulation....PASS
gc_2x25_POPs_BaP_merra2.............................Execute Simulation....PASS
gc_2x25_tagCH4_merra2...............................Execute Simulation....PASS
gc_2x25_tagCO_merra2................................Execute Simulation....PASS
gc_2x25_tagO3_merra2................................Execute Simulation....PASS
gc_2x25_TransportTracers_merra2.....................Execute Simulation....PASS
gc_2x25_TransportTracers_merra2_LuoWd...............Execute Simulation....PASS
gc_4x5_aerosol_geosfp...............................Execute Simulation....PASS
gc_4x5_aerosol_merra2...............................Execute Simulation....PASS
gc_4x5_carbon_geosfp................................Execute Simulation....PASS
gc_4x5_carbon_merra2................................Execute Simulation....PASS
gc_4x5_CH4_geosfp...................................Execute Simulation....PASS
gc_4x5_CH4_merra2...................................Execute Simulation....PASS
gc_4x5_fullchem_aciduptake_merra2...................Execute Simulation....PASS
gc_4x5_fullchem_APM_merra2..........................Execute Simulation....PASS
gc_4x5_fullchem_benchmark_merra2....................Execute Simulation....PASS
gc_4x5_fullchem_complexSOA_merra2...................Execute Simulation....PASS
gc_4x5_fullchem_complexSOA_SVPOA_merra2.............Execute Simulation....PASS
gc_4x5_fullchem_geosfp..............................Execute Simulation....PASS
gc_4x5_fullchem_marinePOA_merra2....................Execute Simulation....PASS
gc_4x5_fullchem_merra2..............................Execute Simulation....PASS
gc_4x5_fullchem_merra2_47L..........................Execute Simulation....PASS
gc_4x5_fullchem_merra2_LuoWd........................Execute Simulation....PASS
gc_4x5_fullchem_RRTMG_merra2........................Execute Simulation....PASS
gc_4x5_fullchem_TOMAS15_merra2_47L..................Execute Simulation....PASS
gc_4x5_fullchem_TOMAS40_merra2_47L..................Execute Simulation....PASS
gc_4x5_Hg_geosfp....................................Execute Simulation....PASS
gc_4x5_Hg_merra2....................................Execute Simulation....PASS
gc_4x5_metals_merra2................................Execute Simulation....PASS
gc_4x5_POPs_BaP_geosfp..............................Execute Simulation....PASS
gc_4x5_POPs_BaP_merra2..............................Execute Simulation....PASS
gc_4x5_tagCH4_geosfp................................Execute Simulation....PASS
gc_4x5_tagCH4_merra2................................Execute Simulation....PASS
gc_4x5_tagCO_geosfp.................................Execute Simulation....PASS
gc_4x5_tagCO_merra2.................................Execute Simulation....PASS
gc_4x5_tagO3_geosfp.................................Execute Simulation....PASS
gc_4x5_tagO3_merra2.................................Execute Simulation....PASS
gc_4x5_TransportTracers_geosfp......................Execute Simulation....PASS
gc_4x5_TransportTracers_geosfp_LuoWd................Execute Simulation....PASS
gc_4x5_TransportTracers_merra2......................Execute Simulation....PASS
gc_4x5_TransportTracers_merra2_LuoWd................Execute Simulation....PASS
 
Summary of test results:
------------------------------------------------------------------------------
Execution tests passed: 51
Execution tests failed: 0
Execution tests not yet completed: 0

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%  All execution tests passed!  %%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```
As did all GCHP integration tests:
==============================================================================
GCHP: Execution Test Results

Number of execution tests: 3
==============================================================================
 
Execution tests:
------------------------------------------------------------------------------
gchp_fullchem_benchmark_merra2_c48..................Execute Simulation....PASS
gchp_fullchem_standard_merra2_c24...................Execute Simulation....PASS
gchp_TransportTracers_geosfp_c24....................Execute Simulation....PASS
 
Summary of execution test results:
------------------------------------------------------------------------------
Execution tests passed:        3
Execution tests failed:        0
Execution tests not completed: 0

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%  All execution tests passed!  %%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

File: run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.aerosol [hunk 1]
```diff
@@ -2442,19 +2442,19 @@ ${RUNDIR_TES_CLIM_N2O}
...
-215 NEI99_DOW_PRPE    $ROOT/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc           PRPE         1999/1-12/WD/0 C xy 1 1
-216 NEI99_DOW_C3H8    $ROOT/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc           C3H8         1999/1-12/WD/0 C xy 1 1
-217 NEI99_DOW_C2H6    $ROOT/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc           C2H6         1999/1-12/WD/0 C xy 1 1
-218 NEI99_DOW_SO2     $ROOT/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc           SO2          1999/1-12/WD/0 C xy 1 1
-219 NEI99_DOW_SO4     $ROOT/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc           SO4          1999/1-12/WD/0 C xy 1 1
-220 NEI99_DOW_MSA     $ROOT/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc           MSA          1999/1-12/WD/0 C xy 1 1
-221 NEI99_DOW_BCPI    $ROOT/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc           BCPI         1999/1-12/WD/0 C xy 1 1
-222 NEI99_DOW_OCPI    $ROOT/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc           OCPI         1999/1-12/WD/0 C xy 1 1
+210 NEI99_DOW_NOX   $ROOT/NEI2005/v2023-02/scaling/NEI99.dow.geos.1x1.corrected.012023.nc NOX  1999/1-12/WD/0 C xy 1 1
```
Comment (msulprizio) [inline run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.aerosol:2445]: The file path here seems to be wrong. There is no scaling/ subdirectory. I think should just be `$ROOT/NEI2005/v2023-02/NEI99.dow.geos.1x1.corrected.012023.nc`.

This needs to be fixed in all modified HEMCO_Config.rc files in this PR.

File: run/GCHP/ExtData.rc.templates/ExtData.rc.carbon [hunk 2]
```diff
@@ -509,6 +509,9 @@ SHIP_LEVS     1 N Y - none none cmv_c3     ./HcoDir/VerticalScaleFactors/v2021-0
 AVIATION_SURF_CORR      1       2004 Y F%y2-01-01T00:00:00  none none CO2 ./HcoDir/CO2/v2022-11/FOSSIL/Aviation_SurfCorr_SclFac.1x1.nc
 OCEANCO2_SCALED_MONTHLY kg/m2/s N    Y F%y2-%m2-01T00:00:00 none none CO2 ./HcoDir/CO2/v2022-11/OCEAN/Scaled_Ocean_CO2_monthly.nc
 
+# --- NEI 2005 scale factors ---
+#NEI99_DOW_CO 1 D N F1999-%m2-01T00:00:00 none none CO ./HcoDir/NEI2005/v2023-02/scaling/NEI99.dow.geos.1x1.corrected.012023.nc
```
Comment (msulprizio) [inline run/GCHP/ExtData.rc.templates/ExtData.rc.carbon:513]: Is this line needed if it's just commented out? It wasn't included in the earlier version.

File: run/GCHP/ExtData.rc.templates/ExtData.rc.fullchem [hunk 3]
```diff
@@ -2151,19 +2151,19 @@ FFF 1 Y Y F2010-%m2-01T00:00:00 none none FFF ./HcoDir/EDGARv43/v2016-11/EDGAR_v
...
-NEI99_DOW_MEK  1 D N F1999-%m2-01T00:00:00 none none MEK  ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_PRPE 1 D N F1999-%m2-01T00:00:00 none none PRPE ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_C3H8 1 D N F1999-%m2-01T00:00:00 none none C3H8 ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_C2H6 1 D N F1999-%m2-01T00:00:00 none none C2H6 ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_SO2  1 D N F1999-%m2-01T00:00:00 none none SO2  ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_SO4  1 D N F1999-%m2-01T00:00:00 none none SO4  ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_MSA  1 D N F1999-%m2-01T00:00:00 none none MSA  ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_BCPI 1 D N F1999-%m2-01T00:00:00 none none BCPI ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_OCPI 1 D N F1999-%m2-01T00:00:00 none none OCPI ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
+NEI99_DOW_NOX  1 D N F1999-%m2-01T00:00:00 none none NOX  ./HcoDir/NEI2005/v2023-02/scaling/NEI99.dow.geos.1x1.corrected.012023.nc
```
Comment (msulprizio) [inline run/GCHP/ExtData.rc.templates/ExtData.rc.fullchem:2154]: File paths also need to be fixed here to remove the scaling/ subdirectory.

**Review by msulprizio — CHANGES_REQUESTED** [review]

**Comment by yantosca** [comment]
Thanks @msulprizio, I'll fix this now.

File: run/GCHP/ExtData.rc.templates/ExtData.rc.carbon (hunk 2 above)
Comment (yantosca) [reply run/GCHP/ExtData.rc.templates/ExtData.rc.carbon:513]: Removed

File: run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.aerosol (hunk 1 above)
Comment (yantosca) [reply run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.aerosol:2445]: Will fix

File: run/GCHP/ExtData.rc.templates/ExtData.rc.fullchem (hunk 3 above)
Comment (yantosca) [reply run/GCHP/ExtData.rc.templates/ExtData.rc.fullchem:2154]: Will fix

**Comment by yantosca** [comment]
Now fixed in commit 227040147

**Review by msulprizio — APPROVED** [review]
Thanks @yantosca. These updates are good to merge.

**Pull request merged**